src/App.js

```diff
@@ -1,12 +1,30 @@
+import React from "react";
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
 import { Header, Footer } from "./components/index";
-import { Home } from "./pages/index";
+import {
+  Home,
+  Login,
+  Signup,
+  ProductListing,
+  ProductDetails,
+  Wishlist,
+  Cart,
+} from "./pages/index";

```

Comment:
You can just give path as ./pages and that would also work, exporting from index.js is automatically judged by React.